python/stressng/README.md

```diff
@@ -0,0 +1,71 @@
+# Official plugins repository for Arcaflow
+
```

Comment:
I don't think we want to keep a license in each plugin and only one common license for all the plugins. please remove this